crates/nu-command/tests/commands/cal.rs

```diff
@@ -33,7 +33,7 @@ fn cal_friday_the_thirteenths_in_2015() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
-        cal --full-year 2015 | default friday 0 | where friday == 13 | length
+        cal --full-year 2015 | default friday '0' | where friday == 13 | length
```

Comment:
I think this test may be wrong. It should be `default 0 friday` (default value column_name). it changed a while back, odd that it works.

I wonder if this incorrect syntax is why we can't default a calendar to a number?